### PR TITLE
Community Partner Highlight - INER

### LINKED ISSUE #96: Community Partner Highlight - INER
This issue tracks the drafting, review and approval for adding a new blog post to the Catalyst Project blog.

This issue is complete when: the copy of this page is approved and published on the website.

**Please edit and insert information enclosed by the <angled brackets> below**

Create a Google Doc in *Catalyst Project Team Drive > External Documentation > Blog posts > your_filename* to draft the copy for your new page, with a title of the form "[VERSION #] <your_title>" and append this title with "(ES)" if the draft is in Spanish.

Please consult the [Style Guide](https://docs.google.com/document/d/1qMSvTmqBuBow98tvxgNvsVZB9eo-pItWfGZPy8W_cok/edit?usp=sharing) for more information, including which metadata tags to choose from for your blog post.

Google doc for drafting copy: https://docs.google.com/document/d/1yEyRCyV9VlpXqiXlJA373VQVIzgnpLAgYzCImPwmp0M/edit

**Roles**

- [ ] Drafting content: @katiepratt 
- [ ] Sub-team review: @jnywong @Tajuddeen1 @LouWoodley 
- [ ] Handoff: @katiepratt to provide editorial review
- [ ] Hand off to Katie for final review @katiepratt 
- [ ] Hand off to MD for translation @SLLDeCuntil the end of April
- [ ] Hand off to Jenny to draft website previews - 2 separate posts @jnywong 
- [ ] Final approval of English post by Katie @katiepratt 
- [ ] Final approval of Spanish post by Sabrina @SLLDeC

### PR DESCRIPTION
**Pull request by jnywong**
Closes #96

Spanish version 👉 https://catalyst-website--104.org.readthedocs.build/en/104/blog/community-highlight-iner-es.html
English version 👉 https://catalyst-website--104.org.readthedocs.build/en/104/blog/community-highlight-iner-en.html

### CODE REVIEW

**Comment by jnywong** [comment]
Suggested edit:
```diff
diff --git a/catalystbook/blog/community-highlight-iner-en.md b/catalystbook/blog/community-highlight-iner-en.md
index 4a03ce9..55526c0 100644
--- a/catalystbook/blog/community-highlight-iner-en.md
+++ b/catalystbook/blog/community-highlight-iner-en.md
@@ -37,4 +37,4 @@ We have been participating in all of the [training](../training.md) offered by C
 A screenshot of the INER-Catalyst Team collaborating remotely thanks to the Catalyst Project.
 :::
 
-**Photo courtesy of Yalbi I. Balderas-Martinez.**
+*Photo courtesy of Yalbi I. Balderas-Martinez.*

```

**Comment by 4iro** [comment]
Suggested edit. Replace image alt text (line 37):
:alt: A screenshot of a Zoom meeting with the INER team.
A screenshot of the INER-Catalyst Team collaborating remotely thanks to the Catalyst Project.
With: 
:alt: Captura de pantalla del equipo INER-Catalyst colaborando de manera remota gracias al Proyecto Catalyst. Foto cortesía de Yalbi I. Balderas-Martinez.

File: catalystbook/blog/community-highlight-iner-es.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+El Proyecto Catalyst nos está permitiendo trabajar de manera más eficiente y colaborativa, y también estamos planeando utilizar la plataforma para implementar talleres de inteligencia artificial.
+
+**¿Podrías contarnos cómo fue hasta ahora tu experiencia con el Proyecto Catalyst? Por ejemplo, ¿has participado en alguna capacitación ofrecida por Catalyst? ¿Qué es lo que has disfrutado más?**
+
+Hemos participado en todas [las capacitaciones](../training.md) ofrecidas por Catalyst, incluidas la formación para Campeones y Campeonas del Hub, la capacitación para Instructores/as de The Carpentries ofrecida por MetaDocencia y el programa OLS Open Seeds.
+
+:::{figure} ../images/blog/community-partner-highlights/iner.png
+:alt: A screenshot of a Zoom meeting with the INER team.
+A screenshot of the INER-Catalyst Team collaborating remotely thanks to the Catalyst Project.
```
Comment (jnywong) [inline catalystbook/blog/community-highlight-iner-es.md:37]:
```suggestion
Captura de pantalla del equipo INER-Catalyst colaborando de manera remota gracias al Proyecto Catalyst.
```

**Comment by jnywong** [comment]
@4iro Thanks for the suggestion! I have applied the change in https://github.com/czi-catalystproject/czi-catalystproject.github.io/pull/104/commits/86d5279f3fb9da33806fed1b31677cbe7806c9f8

**Pull request merged**